views/prisonerProfile/prisonerWorkAndSkills/prisonerWorkAndSkills.njk

```diff
@@ -9,13 +9,20 @@
 {% set errorMessage = "We cannot show these details right now. Try reloading the page." %}
 
 {% set functionalSkillLevels = functionalSkillLevels.content %}
+{% set goals = goals.content %}
```

Comment:
Not sure this is needed, also overloading (reusing) the same variable name is confusing